### ISSUE TITLE
[accessibility] Attribution container should not have `role="list"` (or should have `listitem` children)

### ISSUE THREAD

**Issue by Malvoz** [maplibre/maplibre-gl-js#358]
https://github.com/maplibre/maplibre-gl-js/blob/0c6ead362589465ffa14a8feb99b689640c698d3/src/ui/control/attribution_control.js#L63

The attribution links should either be children of list items ([`role="listitem"`](https://www.w3.org/TR/wai-aria/#listitem)) or [`role="list"`](https://www.w3.org/TR/wai-aria/#list) should be removed.

See motivation in https://dequeuniversity.com/rules/axe/4.2/list?application=RuleDescription (describes `<ul>` and `<li>` requirements, the same is applicable to the equivalent ARIA roles `list` and `listitem`).

<hr>

(The same issue was also reported with MapBox: https://github.com/mapbox/mapbox-gl-js/issues/11033.)
